$MirOS: ports/graphics/tiff/patches/patch-libtiff_Makefile_am,v 1.2 2005/04/24 01:17:36 tg Exp $
--- libtiff/Makefile.am.orig	Tue Jan 11 16:07:06 2005
+++ libtiff/Makefile.am	Sun Apr 24 19:17:42 2005
@@ -107,12 +107,13 @@ endif
 libtiff_la_SOURCES = $(SRCS)
 libtiff_la_LDFLAGS = \
 	-no-undefined \
-	-version-number $(LIBTIFF_MAJOR_VERSION):$(LIBTIFF_MINOR_VERSION):$(LIBTIFF_MICRO_VERSION)
+	-version-number $(LIBTIFF_VERSION_INFO)
 
 libtiffxx_la_SOURCES = $(SRCSXX)
 libtiffxx_la_LDFLAGS = \
 	-no-undefined \
-	-version-number $(LIBTIFF_MAJOR_VERSION):$(LIBTIFF_MINOR_VERSION):$(LIBTIFF_MICRO_VERSION)
+	-version-number $(LIBTIFF_VERSION_INFO)
+libtiffxx_la_LIBADD = libtiff.la
 
 #
 # The finite state machine tables used by the G3/G4 decoders
